packages/graphql/src/classes/Executor.ts

```diff
@@ -82,6 +82,7 @@ export type ExecutorConstructorParam = {
     executionContext: ExecutionContext;
     cypherQueryOptions?: CypherQueryOptions;
     sessionConfig?: SessionConfig;
+    cypherParams?: Record<string, any>;
```

Comment:
Does this need to be `Record<string, any>`? Could `Record<string, unknown>` work here?